monkey/monkey_island/cc/ui/src/components/configuration-components/CheckboxWithMessage.js

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExclamationTriangle } from '@fortawesome/free-solid-svg-icons/faExclamationTriangle';
+
+class CheckboxWithMessage extends React.Component {
+
+  render() {
+    let warningMessageComponent = <></>;
+    let infoMessageComponent = <></>;
+
+    if (Object.prototype.hasOwnProperty.call(this.props.schema, 'warning_message')) {
+      warningMessageComponent = (
+        <div className='alert alert-warning'>
+          <FontAwesomeIcon icon={faExclamationTriangle} style={{ 'marginRight': '5px' }} />
+          {this.props.schema.warning_message}
+        </div>
+      );
+    }
+
+    if (Object.prototype.hasOwnProperty.call(this.props.schema, 'info_message')) {
+      infoMessageComponent = (
+        <div className='alert alert-info'>
+          <FontAwesomeIcon icon={faExclamationTriangle} style={{ 'marginRight': '5px' }} />
+          {this.props.schema.info_message}
+        </div>
+      );
+    }
```

Comment:
Does it make sense to put `InfoBox.js` and `WarningBox.js` back and just have these `if` statements create InfoBox and WarningBox components?